alerts/jira.go

```diff
@@ -194,7 +195,7 @@ func getSummary(ctx AlertContext) string {
 // historyURL provides a full URL to the test's task history page.
 func historyURL(t *task.Task, testName, uiRoot string) string {
 	return fmt.Sprintf("%v/task_history/%v/%v#%v=fail",
-		uiRoot, t.Project, t.DisplayName, testName)
+		uiRoot, t.Project, url.PathEscape(t.Id), testName)
```

Comment:
I believe this could be url.Parse(fmt.Sprintf(...)).String(), it would definitely be slower but probably more resilient to changes.

Reply:
historyURL is also in trigger/task_jira.go, so I'll need you to do the same there. (sorry we're in the middle of deprecating/refactoring this code/system, hence the duplication).

Reply:
No worries, should have figured anyway.